src/gurt/dlog.c

```diff
@@ -636,7 +637,10 @@ void d_vlog(int flags, const char *fmt, va_list ap)
 	 * NB: flush to logfile if the message is important (warning/error...)
 	 * or the last flush was 1+ second ago.
 	 */
```

Comment:
Need top change comment.

Reply:
the comment is still mostly valid (importance now is just not hardcoded). i ll update it on next pr to not consume test resources for this